Cargo-minimal.lock

```diff
@@ -10,9 +10,9 @@ checksum = "dde20b3d026af13f561bdd0f15edf01fc734f0dafcedbaf42bba506a9517f223"
 
 [[package]]
 name = "arrayvec"
-version = "0.7.4"
+version = "0.7.6"
```

Comment:
In 51fd4e86b51a199b8553e6f32295b0ab9c042307:

I'm struggling to understand this change. `cargo update -Z direct-minimal-versions` consistently makes it, but the actual version of arrayvec that we depend on is 0.7.2 (defined by hex-conservative --- this is not even a direct dependency!). I'm not sure why this was at 0.7.4 instead of 0.7.2 and I really don't understand why it's now 0.7.6.

Reply:
I believe another example of a transitive dep version bump since `arrayvec` is transitive for this workspace. hex-conservative sets [`arrayvec = { version = "0.7.2", default-features = false }`](https://github.com/rust-bitcoin/hex-conservative/blob/master/Cargo.toml#L33). I'll double check when using this syntax, but I think this is equivalent to the `^0.7.2` notation, so minor and patch versions will be updated.

I am not sure why the `0.7.4` version was set before, but [looks like 0.7.6 is the highest version which fits the constraints.](https://crates.io/crates/arrayvec/versions)